config/linuxsuren/http-downloader.yml

```diff
@@ -1,2 +1,5 @@
-filename: "hd-{{.OS}}-{{.Arch}}.tar.gz"
+filename: "hd-{{.OS}}-{{.Arch}}"
 binary: hd
+formatOverrides:
+  windows: zip
+  linux: tar.gz
```

Comment:
Normally, we suggest an empty line at the end of each file. See also [the explanation](https://stackoverflow.com/questions/2287967/why-is-it-recommended-to-have-empty-line-in-the-end-of-a-source-file).

Reply:
Thanks for the comment. Made the change!